modules/banner/react/stories/stories.tsx

```diff
@@ -7,7 +7,7 @@ import withReadme from 'storybook-readme/with-readme';
 import Banner from '../index';
 import README from '../README.md';
 
-const handleBannerClick = (e: React.SyntheticEvent) => {
+const handleBannerClick = () => {
```

Comment:
We weren't using the event here, so I removed it. I can add it back for documentation purposes if we'd like though.

Reply:
I don't think it is necessary.

Private functions like this aren't in documentation anywhere. If the component has a specific interface, you'll have to set this correctly to assign it to the Component's prop list.